src/go/rdctl/pkg/utils/utils.go

```diff
@@ -0,0 +1,77 @@
+package utils
+
+import (
+	"io/fs"
+	"os"
+	"path"
+
+	"github.com/rancher-sandbox/rancher-desktop/src/go/rdctl/pkg/directories"
+)
+
+// Get the parent (or grandparent, or great-grandparent...) directory of fullPath.
+// numberTimes is the number of steps to ascend in the directory hierarchy.
+func MoveToParent(fullPath string, numberTimes int) string {
+	fullPath = path.Clean(fullPath)
+	for ; numberTimes > 0; numberTimes-- {
+		fullPath = path.Dir(fullPath)
+	}
+	return fullPath
+}
```

Comment:
Shouldn't this function have some kind of error handling. E.g. does it make sense to return `.` as the result of `MoveToParent("/a/b/c", 5)`?

Reply:
I also find the name `MoveToParent` misleading, as it sounds like it is changing directories, or something. Maybe call it `ParentDir` or something that sounds more like a query and not an action?

Reply:
I agree 100%. This code was taken from cmd/start.go, and I was trying to limit the number of changes that were made to it so this PR didn't grow in scope. In retrospect, we probably shouldn't have pulled in that code to begin with, and then dealt with the duplication in a subsequent PR. I'm not sure how to proceed now...

Reply:
There's also https://github.com/rancher-sandbox/rancher-desktop/pull/3798#discussion_r1103104661

Reply:
Given that it is copied code, let's not expand the scope of this PR, but create a new issue with all the refactoring you think should be done.

I would put it into "Later", but if you still have available time in this release cycle, feel free to start on it. There may be more important issues in the backlog though.